src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+        System.out.println(ball2);
+        System.out.println(ball3);
+        System.out.println(ball1
```

Comment:
It seems you have accidentally placed the closing parenthesis and semicolon on a new line. This is not a common practice and can make the code less readable. It's better to keep the closing parenthesis and semicolon on the same line as the method call.